public/data/MasterReach1.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "MasterReach1",
+  "type": "personal",
+  "bio": "Web Developer | Ethical Hacker | Open Source Enthusiast",
+  "avatar": "https://github.com/MasterReach1.png",
+  "links": [
+    {
+      "name": "Follow me here on Github!",
+      "url": "https://github.com/MasterReach1",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter for tech info, development updates, and some meme content!",
+      "url": "https://twitter.com/findevxone",
+      "icon": "twitter"
+    }
+   ],
+  "milestones": [
+    {
+      "title": "Started developing games with Python & Blender",
+      "date": "October 2013",
+      "icon": "book",
+      "color": "orange",
+      "description": "Started with Python",
```

Please remove this trailing comma from line no. `24`. 
```suggestion
      "description": "Started with Python"
```

---

I have made this change. Do I need to make another PR or can the changes be seen on this one?

I'm super new to this, so not 100% on how it works.

---

This current PR is fine. You don't need to make another PR to make changes in this PR. 😃